src/components/TabletApp/LeftSide/UpcomingCards/UpcomingCards.tsx

```diff
@@ -5,6 +5,7 @@ import SIZE from "../../../../constants/CustomSize";
 import { useNavigate } from "react-router";
 import COLORS from "../../../../constants/CustomColors";
 import { StringDecoder } from "string_decoder";
+import { useLocation } from "react-router-dom";
```

Comment:
Maybe rearrange the imports,  the react-routers together, and the COLORS import separate from the rest as it's a custom import